mvpa2/clfs/dummies.py

```diff
@@ -52,7 +52,7 @@ class RandomClassifier(Classifier):
     """Dummy classifier deciding on labels absolutely randomly
     """
 
-    __tags__ = ['random', 'non-deterministic']
+    __tags__ = ['random', 'non-deterministic', 'one-class-binary']
```

Comment:
I wonder though why I didn't give it even "binary" or "mutliclass" originally... hm...  probably so it doesn't even participate in any case where meaningful classifier is requested (e.g. in tests), so probably not worth adding oneclass tags here either